src/Synth/LFO.cpp

```diff
@@ -289,7 +289,7 @@ float LFO::lfoout()
         computeNextFreqRnd();
     }
             
-    float watch_data[2] = {phase, out};
+    float watch_data[2] = {phaseWithStartphase, out};
```

Comment:
This will now take the phase without the increase from the previous lines. I think that's slightly wrong.

Reply:
it makes more sense, that watch shows the phase that was used to calculate out. So in a way it was wrong before.